server/models/DeviceWearer.ts

```diff
@@ -16,7 +16,7 @@ const DeviceWearerModel = z.object({
     .string()
     .nullable()
     .transform(val => (val === null ? [] : val.split(','))),
-  noFixedAbode: z.boolean().nullable(),
+  noFixedAbode: z.boolean().nullish(),
```

Comment:
This can only be `null`, `true` or `false`. Never `undefined`

Reply:
again when I was running the API it would not validate. I will reload the container and see if that allows me to revert this and similar changes

Reply:
reverted